prov/efa/src/efa_fabric.c

```diff
@@ -899,6 +899,43 @@ static struct fi_ops_fabric efa_ops_fabric = {
 	.trywait = ofi_trywait
 };
 
+static
+void efa_atfork_callback()
+{
+	static int visited = 0;
+
+	if (visited)
+		return;
+
+	visited = 1;
+	if (getenv("RDMAV_FORK_SAFE") || getenv("IBV_FORK_SAFE") )
+		return;
+
+	fprintf(stderr,
+		"A process has executed an operation involving a call\n"
+		"to the fork() system call to create a child process.\n"
+		"\n"
+		"As a result, the libfabric EFA provider is operating in\n"
+		"a condition that could result in memory corruption or\n"
+		"other system errors.\n"
+		"\n"
+		"For the libfabric EFA provider to work safely when fork()\n"
+		"is called, you will need to set the following environment\n"
+		"variable:\n"
+		"          RDMAV_FORK_SAFE\n"
+		"\n"
+		"However, setting this environment variable can result in\n"
+		"signficant performance impact to your application due to\n"
+		"increased cost of memory registration.\n"
+		"\n"
+		"You may want to check with your application vendor to see\n"
+		"if an application-level alternative (of not using fork)\n"
+		"exists.\n"
+		"\n"
+		"You job is aborted.\n");
```

Comment:
Typo and nit: `You job is aborted` -> `Your job will now abort.`

Reply:
Thank you! fixed in new revision.

Reply:
fixed the typo again.